packages/api/lib/rulesHelpers.js

```diff
@@ -492,18 +493,18 @@ async function addRule(item, payload) {
  * @returns {void}       - Returns if record is valid, throws error otherwise
  */
 function recordIsValid(rule) {
-  const error = new Error('The record has validation errors');
+  const error = new Error('The record has validation errors. ');
   error.name = 'SchemaValidationError';
   if (!rule.name) {
-    error.detail = 'Rule name is undefined.';
+    error.message += 'Rule name is undefined.';
     throw error;
   }
   if (!rule.workflow) {
-    error.detail = 'Rule workflow is undefined.';
+    error.message += 'Rule workflow is undefined.';
     throw error;
   }
   if (!rule.rule.type) {
-    error.detail = 'Rule type is undefined.';
+    error.message += 'Rule type is undefined.';
```

Comment:
I changed these because POST doesn't throw the error like PUT does, it returns a response with `error.message` https://github.com/nasa/cumulus/pull/2728/files#diff-907b67e0c8c9667871fad5f88fa710ae9231500c156a692721ff54c84254b7b9R115

Reply:
if the `POST` returns a response with `error.message`, Does this cause an issue with `PUT` since this is a change in behavior?

Reply:
Nevermind. I see the answer to my question in `test-rules.js`